### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by SinuheM**
**TRELLO TICKET**: https://trello.com/c/Cx8TaJDM

**Description**: Functionality for the footer component

Desktop view
image

Mobile view
image

### CODE REVIEW

**Review by coding-tomato — COMMENTED** [review]
Looks good to me!

**Review by coding-tomato — APPROVED** [review]
Looks good to me!

**Comment by davidvm1** [comment]
Please rebase and squash your commits before sending the PR.

File: styles/globals.scss [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 @import "./base/base";
 @import "./base/typography";
+@import "./components/navigation";
+@import "./components/stamp";
+@import "./components/socialMedia";
+@import "./components/sitemap";
 @import "./components/title";
 @import "./components/subtitle";
+@import "./layout/container";
+@import "./layout/footer";
```
Comment (davidvm1) [inline styles/globals.scss]: To import the components is inefficient to import one by one, better create an expression for importing all files inside components instead.

File: styles/globals.scss [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
 body {
```
Comment (davidvm1) [inline styles/globals.scss]: Remember that base styles for tags should be in the base.scss not in the globals, globals should only have the imports

File: utils/contentful.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
 const dataContentful = {};
-export default function headerContentful(data) {
```
Comment (davidvm1) [inline utils/contentful.js]: getHeaderContenfulData would be a more precise name for the method

File: components/Navigation.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { useState } from 'react';
-import headerContentful from '../utils/contentful';
+import { headerContentful } from '../utils/contentful';
```
Comment (dyepes62) [inline components/Navigation.js]: I think this import will change for Orlando in another branch

**Comment by davidvm1** [comment]
Also, there are conflicts for the merge, be carefull with that. Make sure you pull the most recent changes from develop to the branch

File: styles/globals.scss (hunk 1 above)
Comment (SinuheM) [reply styles/globals.scss]: Could you give me more details please

**Review by dyepes62 — APPROVED** [review]

**Pull request merged**